source/frontend/src/features/disposition/list/DispositionSearchResults/columns.tsx

```diff
@@ -88,37 +85,33 @@ export const columns: ColumnWithProps<DispositionSearchResultModel>[] = [
     maxWidth: 40,
     Cell: (props: CellProps<DispositionSearchResultModel>) => {
       const team = props.row.original.dispositionTeam;
-      const personsInTeam = team?.filter(x => x.personId !== undefined);
-      const organizationsInTeam = team?.filter(x => x.organizationId !== undefined);
+      const personsInTeam = team?.filter(x => isValidId(x.personId));
+      const organizationsInTeam = team?.filter(x => isValidId(x.organizationId));
 
       const personsAsString: MemberRoleGroup[] = chain(personsInTeam)
-        .groupBy((groupedTeams: Api_DispositionFileTeam) => groupedTeams.personId)
-        .map<MemberRoleGroup>(x => {
-          return {
-            id: x[0].id?.toString() || '',
-            person: x[0].person || {},
-            organization: null,
-            roles: x
-              .map(t => t.teamProfileType)
-              .filter((z): z is Api_TypeCode<string> => z !== undefined)
-              .flatMap(y => y.description || ''),
-          };
-        })
+        .groupBy((groupedTeams: ApiGen_Concepts_DispositionFileTeam) => groupedTeams.personId)
+        .map<MemberRoleGroup>(x => ({
```

Comment:
oh boy, this looks sketchy, this must have been missed in review. Not needing to be corrected in your PR.